docs/blog/articles/2024-12-13-KServe-0.14-release.md

```diff
@@ -0,0 +1,149 @@
+
+The unstable situation has been mitigated by configuring the OCI model as an init container in addition to also configuring it as a sidecar. The configuration as an init container ensures that the model is fetched before the main containers are started. The prefetching allows the modelcar to start quickly.
+The stabilization is available since KServe version 0.14, where modelcars are now a stable feature.
+
+### Future plan
+
+Modelcars is one implementation option for supporting OCI images for model storage. There are other alternatives commented in [issue #4083](https://github.com/kserve/kserve/issues/4083).
+
+Using volume mounts based on OCI artifacts is the optimal implementation, but this is only [recently possible since Kubernetes 1.31](https://kubernetes.io/blog/2024/08/16/kubernetes-1-31-image-volume-source/) as a native alpha feature. KServe can now evolve to use this new Kubernetes feature.
+
+## Introducing model cache
+
+With models increasing in size, specially true for LLM models, pulling from storage each time a pod is created can result in unmanageable start-up times. Although OCI storage also has the benefit of model caching, the capabilities are not flexible since the management is delegated to the cluster.
+
+The Model Cache was proposed as another alternative to enhance KServe usability with big models, released in KServe v0.14 as an **alpha** feature. It relies on a PV for storing models and it provides control about which models to store in the cache. The feature was designed to mainly to use node Filesystem as storage. Read the [design document for the details](https://docs.google.com/document/d/1nao8Ws3tonO2zNAzdmXTYa0hECZNoP2SV_z9Zg0PzLA/edit).
+
+The model cache is currently disabled by default. To enable, you need to modify the `localmodel.enabled` field on the `inferenceservice-config` ConfigMap.
+
+You start by creating a node group as follows:
+
+```yaml
+apiVersion: serving.kserve.io/v1alpha1
+kind: LocalModelNodeGroup 
+metadata:
+  name: nodegroup1
+spec:
+  persistentVolumeSpec:
+    accessModes:
+    - ReadWriteOnce
+    volumeMode: Filesystem
+    capacity:
+      storage: 2Gi
+    hostPath:
+      path: /models
+      type: ""
+    persistentVolumeReclaimPolicy: Delete
+    storageClassName: standard
+  persistentVolumeClaimSpec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 2Gi
+    storageClassName: standard
+    volumeMode: Filesystem
+    volumeName: kserve
+
+```
+
+Then, you can specify to store an cache a model with the following resource:
+
+```yaml
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterLocalModel
+metadata:
+  name: iris
+spec:
+  modelSize: 1Gi
+  nodeGroup: nodegroup1
+  sourceModelUri: gs://kfserving-examples/models/sklearn/1.0/model
+```
```

Comment:
maybe add the isvc with the example explaining how to use this

Reply:
AFAIK, the InferenceService doesn't change and you use it normally (i.e. you still would use `gs://kfserving-examples/models/sklearn/1.0/model` for `storageUri`) .

The difference you would notice is that the model will be fetched/mounted from the cache, instead of downloading it.

Reply:
I think I can add a brief note about what I just wrote.